### DISCUSSION TITLE
Support for EdgeDb - next generation SQL

### DISCUSSION THREAD

**Discussion by bernardro** [directus/directus#13728] [Feature Requests]
EdgeDb represents the next generation of Relational database that empowers developers to
- reduce cognitive load when navigating between database and code
- build safer and more maintainable software
- build more efficient software

**Comment by bernardro** [comment]
it sits on top of postgres so perhaps we may be able to get this for free?

**Comment by rijkvanzanten** [comment]
It looks like a bespoke Graph style database that's built on top of Postgres, I don't think this is something that's trivial to support, as it doesn't seem to be actually Postgres compatible

**Reply by bernardro** [reply]
They do say that it "plays nice with today's languages"
which I interpret to mean - it has language bindings that are intuitive and easy to integrate into other software

not sure if that makes a difference

**Reply by rijkvanzanten** [reply]
Unfortunately not.. We're currently generating SQL statements in our abstraction in the APIs, which is why we're currently compatible with all the SQL databases that we are. Seeing that EdgeDB doesn't actually support SQL, it'd mean we'd have to come up with a totally new data abstraction that works with their proprietary ORM style interface for querying data, which is a ton of work.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
